src/components/Todo.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { FC } from "react";
+
+import { TodoInterface } from "types/todos";
+
+import { StyleSheet, css } from "aphrodite";
+import { colors, typography } from "styles";
+import { i18n } from "locale";
+
+const Todo: FC<TodoInterface> = ({ id, text }) => {
+  const handleClick = (): void => {
+    alert(i18n.t("test:alertTodo"));
+  };
+  return (
+    <>
+      <div
+        className={css(styles.task)}
+        onClick={() => {
+          handleClick();
+        }}
+        onKeyDown={handleClick}
+        role="menuitem"
+        tabIndex={Math.random()}
```

Comment:
Tell me more about those 3 props

Reply:
Eslint wanted to set role on this element, but we can not assign role="button" to div , so I chose role, for me the best for that component role="menuitem". This role need tabIndex, the eslint said that is important for people which only use keyboard. But if we will be have in this component two buttons in future, it will be deleted.